lib/data/data.dart

```diff
@@ -6,6 +8,79 @@ class UserDetail {
   UserDetail({required this.uid, this.phone, this.email});
 }
 
+enum PropertyTypes {
+  house,
+  apartment;
+
+  static PropertyTypes fromName(String name) =>
+      values.firstWhere((element) => element.name == name);
```

Comment:
firstWhere should have orElse to assign default value when no matches value found, to avoid app crash